compiler/rustc_mir_build/src/thir/pattern/const_to_pat.rs

```diff
@@ -390,11 +390,14 @@ impl<'a, 'tcx> ConstToPat<'a, 'tcx> {
                 ty::Slice(elem_ty) if elem_ty == tcx.types.u8 => PatKind::Constant { value: cv },
```

Comment:
Why are we still converting some slices to `PatKind::Constant`? Aren't the only constants left `&str` and opaque ones?

Reply:
You're looking on something that is changed in a later commit

Reply:
Oops sorry